detour/query.go

```diff
@@ -2048,7 +2054,7 @@ func (q *NavMeshQuery) FinalizeSlicedFindPath(path []PolyRef, maxPath int) (path
 		return 0, Failure
 	}
 
-	var n int
```

Comment:
Variables are already set to their zero values in Go, so this is not really useful